notebook/static/base/js/namespace.js

```diff
@@ -1,83 +1,83 @@
-// Copyright (c) Jupyter Development Team.
-// Distributed under the terms of the Modified BSD License.
-
-
```

Comment:
Why is this entire file showing up as changed? Can you do it without this?

Reply:
As far as I can see there are no changes. I have no idea why this has happened